docs/sdk/client-side-sdks/react-native/react-native.md

```diff
@@ -31,3 +37,25 @@ The SDK is available as a package on npm. It is also open source and can be view
 ## Requirements: 
 
 This SDK is compatible with _React Native_ version 0.64.0 and above.
+
+## React Native Web
+
+To get your React Native Web working with DevCycle, you will need to change one of the rules in the webpack config to include `.cjs` files as one of the file types to be transpiled, e.g.:
```

Comment:
can you link the same github issue we link to in React docs

Reply:
https://github.com/facebook/create-react-app/pull/12021#issuecomment-1108426483 this one?

Reply:
ya

Reply:
It's been updated 👍🏼